snippet-extractor-output/snippets.yaml

```diff
@@ -482,7 +482,7 @@ Excel.CustomXmlPartScopedCollection.getItemOrNullObject:
 
         await context.sync();
     });
-Excel.CustomXmlPartScopedCollection.getByNamespace:
```

Comment:
Maybe this should be a general example of Excel.CustomXmlPartScopedCollection (not an individual method).

Reply:
the property getByNamespace is of type Excel.CustomXmlPartScopedCollection but belongs to Excel.CustomXmlPartCollection which is why I've set it like that. If anything, there could be an argument for setting the example at Excel.CustomXmlPartCollection level.

Reply:
Right. I guess my thought was this snippet is "some stuff done to a scoped collection", so it should be there. I don't have a strong opinion, so I trust your judgement.

Reply:
Actually, I should say it returns a Scoped one.